docker/Dockerfile.hsm

```
@@ -1,4 +1,4 @@
-FROM golang AS builder
+FROM golang:bullseye AS builder
```

The de facto `golang` will use the latest Debian, and we'd like to keep it this way.

```suggestion
FROM golang AS builder
```